tim1/src/main/java/isamrs/tim1/model/Airline.java

```diff
@@ -50,6 +52,10 @@ public Airline() {
 		normalReservations = new HashSet<FlightReservation>();
 	}
 
+	public Airline(ServiceDTO airline) {
+		super(airline);
```

Comment:
It is needed to add plane segments to airline (first, business and economy class)